src/test/HostActivationTests/NativeHosting/Nethost.cs

```diff
@@ -13,6 +13,7 @@ public class Nethost : IClassFixture<Nethost.SharedTestState>
     {
         private const string GetHostFxrPath = "get_hostfxr_path";
         private const int CoreHostLibMissingFailure = unchecked((int)0x80008083);
```

Comment:
Nit: Maybe create a new class under `Constants` and start populating it with error codes... we have quite a few places where we compare these, eventually would be nice to have constants for them